django/db/migrations/questioner.py

```diff
@@ -188,6 +192,24 @@ def ask_rename(self, model_name, old_name, new_name, field_instance):
         return self._boolean_input(msg % (model_name, old_name, model_name, new_name,
                                           field_instance.__class__.__name__), False)
 
+
+    def ask_small_to_big_autofield(self, field_name, model_name):
+        """Adding an auto_now_add field to a model."""
+        if not self.dry_run:
+            choice = self._choice_input(
+                "You have upgraded Django, which will change the field {}.{} to a BigAutoField'. "
```

Comment:
Please rephrase in a way which will stay valid for the case where the change is made independently of the upgrade.
Something like  `"You are changing the field {}.{} from a SmallAutoField to a BigAutofield (perhaps as a result of upgrading Django)"`